Comparing source compatibility of /home/jenkins/.m2/repository/org/apache/jspwiki/jspwiki-kendra-searchprovider/2.11.3-SNAPSHOT/jspwiki-kendra-searchprovider-2.11.3-SNAPSHOT.jar against /home/jenkins/.m2/repository/org/apache/jspwiki/jspwiki-kendra-searchprovider/2.11.0/jspwiki-kendra-searchprovider-2.11.0.jar
WARNING: You are using the option '--ignore-missing-classes', i.e. superclasses and interfaces that could not be found on the classpath are ignored. Hence changes caused by these superclasses and interfaces are not reflected in the output.
===  UNCHANGED CLASS: PUBLIC org.apache.wiki.search.kendra.KendraSearchProvider  (not serializable)
	===  CLASS FILE FORMAT VERSION: 52.0 <- 52.0
	===  UNCHANGED SUPERCLASS: java.lang.Object (<- java.lang.Object)
	===  UNCHANGED CONSTRUCTOR: PUBLIC KendraSearchProvider()
	===  UNCHANGED METHOD: PROTECTED com.amazonaws.services.kendra.AWSkendra buildClient()
	===  UNCHANGED METHOD: PUBLIC java.util.Collection findPages(java.lang.String, org.apache.wiki.api.core.Context)
		===  UNCHANGED EXCEPTION: org.apache.wiki.api.exceptions.ProviderException
		===  UNCHANGED EXCEPTION: java.io.IOException
	===  UNCHANGED METHOD: PUBLIC java.lang.String getDataSourceName()
	===  UNCHANGED METHOD: PUBLIC java.lang.String getIndexName()
	===  UNCHANGED METHOD: PUBLIC com.amazonaws.services.kendra.AWSkendra getKendra()
	===  UNCHANGED METHOD: PUBLIC java.lang.String getProviderInfo()
	===  UNCHANGED METHOD: PUBLIC void initialize(org.apache.wiki.api.core.Engine, java.util.Properties)
		===  UNCHANGED EXCEPTION: org.apache.wiki.api.exceptions.NoRequiredPropertyException
		===  UNCHANGED EXCEPTION: java.io.IOException
	===  UNCHANGED METHOD: PUBLIC void initializeIndexAndDataSource()
	===  UNCHANGED METHOD: PUBLIC void pageRemoved(org.apache.wiki.api.core.Page)
	===  UNCHANGED METHOD: PUBLIC void reindexPage(org.apache.wiki.api.core.Page)
	===  UNCHANGED METHOD: PUBLIC void setKendra(com.amazonaws.services.kendra.AWSkendra)
